mimic/model/nova_objects.py

```diff
@@ -927,6 +940,67 @@ def request_action(self, http_action_request, server_id, absolutize_url):
         else:
             return dumps(bad_request("There is no such action currently supported", http_action_request))
 
+    def _create_list_of_flavors(self):
+        """
+        Creates a list of flavors and stores them in the flavors store
+        """
+        flavors = [StandardFlavor, ComputeFlavor, Performance1Flavor, OnMetalFlavor,
+                   Performance2Flavor, MemoryFlavor, IOFlavor, GeneralFlavor]
```

Comment:
Creating an array of classes and passing them to the _generate_flavors_info method seems clumsy to me.  Any suggestion for a more elegant solution?
@glyph